app/models/rules/challenge.rb

```diff
@@ -0,0 +1,44 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+class Rules::Challenge < Rule
+  def variable_requirement?
+    true
+  end
+
+  def available_challenges
+    [
+      'No Income/Set Income',
+      'Domestic Violence',
+      'No rental history',
+      'Bad Credit',
+      'Eviction History',
+      'Misdemeanor',
+      'Felony Conviction /Criminal History',
+      'Sex Offender',
+      'Meth production conviction',
```

Comment:
The inconsistent capitalization worries me that they may change it in the assessment. Maybe title case them and down case the json values? (There is a note for that on the Warehouse side as well).